lib/matplotlib/rcsetup.py

```diff
@@ -501,6 +501,8 @@ def __call__(self, s):
 
 # a map from key -> value, converter
 defaultParams = {
+    'style': ['', validate_stringlist],
```

Comment:
As mentioned in the original PR, I don't think it's important to support a list of parent stylesheets---at least not initially. I could be convinced otherwise, but I think it adds complication to the logic.